src/utils.ts

```diff
@@ -19,37 +19,52 @@ import { bytesToHex, hexToBytes } from '@noble/curves/abstract/utils';
 import { sha256 } from '@noble/hashes/sha256';
 import { decodeCBOR, encodeCBOR } from './cbor.js';
 
-function splitAmount(value: number, amountPreference?: Array<AmountPreference>): Array<number> {
+function splitAmount(
+	value: number,
+	keyset: Keys,
+	amountPreference?: Array<AmountPreference>,
+	order?: string
```

Comment:
It looks like order is not really used yet, and if it will be used later, typing it as string is a bit ambiguous to me
```suggestion
	isDesc?: boolean
```